PSMDB-1487 Docs for 8.0

nastena1606 opened the pull request.

nastena1606 committed:
PSMDB-1487 Docs for 8.0

radoslawszulgo commented:
Change in docs/index.md missing - it still displays 7.0

nastena1606 committed:
Added instructions to disable glibc for tarballs and enable THP for upgrade

nastena1606 committed:
PSMDB-1519 Updated the URL for Pro tarballs (#919)

modified:   docs/install/install-pro.md

---------

Co-authored-by: Sandra Romanchenko <53295797+sandraromanchenko@users.noreply.github.com>

nastena1606 committed:
Further doc updates

nastena1606 committed:
Updated version to 8.0.1

sandraromanchenko commented on docs/install/tarball.md:
```diff
@@ -1,10 +1,10 @@
 # Install Percona Server for MongoDB from binary tarball
 
-You can find links to the binary tarballs under the *Generic Linux* menu item on the [Percona website](https://www.percona.com/downloads/percona-server-mongodb-7.0/)
+You can find links to the binary tarballs under the *Generic Linux* menu item on the [Percona website](https://www.percona.com/downloads/percona-server-mongodb-6.0/)
```
```suggestion
You can find links to the binary tarballs under the *Generic Linux* menu item on the [Percona website](https://www.percona.com/downloads/percona-server-mongodb-8.0/)
```

sandraromanchenko commented on mkdocs-base.yml:
```diff
@@ -128,17 +129,22 @@ plugins:
-          cover_subtitle: 7.0.12-7 (July 23, 2024)
+          output_path: '_pdf/PerconaServerforMongoDB-8.0.pdf'
+          cover_title: 'Server for MongoDB Documentation 8.0'
+          cover_subtitle: 
```
```suggestion
          cover_subtitle: 8.0.1-1 (October 30, 2024)
```

sandraromanchenko commented on docs/psmdb-pro.md:
```diff
@@ -2,7 +2,7 @@
 Percona Server for MongoDB Pro is a build of Percona Server for MongoDB that contains purpose-built enterprise [features](#features). It is wrapped in packages created and tested by Percona and is available exclusively for Percona customers.
 
-Percona Server for MongpDB Pro is available starting with version [7.0.4-2](release_notes/7.0.4-2.md).
+Percona Server for MongpDB Pro is available starting with version [7.0.4-2](release_notes/https://docs.percona.com/percona-server-for-mongodb/release_notes/7.0.4-2.html).
```
```suggestion
Percona Server for MongoDB Pro is available starting with version [8.0.1-1](release_notes/https://docs.percona.com/percona-server-for-mongodb/release_notes/8.0.1-1.html).
```

sandraromanchenko commented on docs/psmdb-pro.md:
```diff
@@ -14,7 +14,7 @@ Find the list of solutions available in Percona Server for MongoDB Pro builds:
 | Name                                | Version added | Description  | 
 | ----------------------------------- | ------------- | -------------
-| [FIPS support ](fips.md)| [7.0.4-2](release_notes/7.0.4-2.md) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
+| [FIPS support ](fips.md)| [7.0.4-2](https://docs.percona.com/percona-server-for-mongodb/release_notes/7.0.4-2.html) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
```
```suggestion
| [FIPS support ](fips.md)| [8.0.1-1](https://docs.percona.com/percona-server-for-mongodb/release_notes/8.0.1-1.html) | FIPS mode provides a way to use FIPS-compliant encryption and run the Percona Server for MongoDB with the FIPS-140 certified library for OpenSSL. This helps customers meet minimum security requirements for cryptographic modules and testing in both hardware and software. |
```

nastena1606 committed:
Removed reference for non-existing variable

nastena1606 committed:
Removed confusing version from LDAP auth doc

nastena1606 commented on docs/install/source.md:
````diff
@@ -35,43 +36,56 @@ To build Percona Server for MongoDB manually, you need the following:
 
     ```{.bash data-prompt="$"}
-    $ sudo yum -y install gcc gcc-c++ cmake curl openssl-devel openldap-devel krb5-devel libcurl-devel cyrus-sasl-devel bzip2-devel zlib-devel lz4-devel xz-devel e2fsprogs-devel
+    $ gcc g++ cmake curl libssl-dev libldap2-dev libkrb5-dev libcurl4-openssl-dev libsasl2-dev liblz4-dev libpcap-dev libbz2-dev libsnappy-dev zlib1g-dev libzlcore-dev liblzma-dev ibext2fs-dev e2fslibs-dev bear
````
```suggestion
    $ sudo apt install -y gcc g++ cmake curl libssl-dev libldap2-dev libkrb5-dev libcurl4-openssl-dev libsasl2-dev liblz4-dev libpcap-dev libbz2-dev libsnappy-dev zlib1g-dev libzlcore-dev liblzma-dev ibext2fs-dev e2fslibs-dev bear
```

nastena1606 committed:
Aligned build from source with Contrib guide

nastena1606 committed:
Fixed prompt for build commands, added missing Python package

radoslawszulgo reviewed (changes requested):
Please remove distro ;)

nastena1606 committed:
PSMDB-1539 Updated upgrade instuctions to drop configuration of fork and pidFilePath options

	modified:   docs/install/upgrade-from-70.md
	modified:   docs/install/upgrade-from-mongodb.md

nastena1606 committed:
Updated version and the date

nastena1606 commented:
> Please remove distro ;)

@radoslawszulgo we don't list distro in this doc set

nastena1606 committed:
Merge branch '8.0' into PSMDB-1487-Docs-8.0
